django/db/models/aggregates.py

```diff
@@ -20,6 +20,7 @@ class Aggregate(Func):
     filter_template = '%s FILTER (WHERE %%(filter)s)'
     window_compatible = True
     allow_distinct = False
+    empty_aggregate_value = None
```

Comment:
I was just thinking that perhaps we should name this `empty_value` instead:

```suggestion
    empty_value = None
```

After all, this attribute is intended for use on subclasses of `Aggregate` only, so the naming is a bit redundant.

Reply:
@pope1ni it's actually defined as a property on `BaseExpression`, which `Aggregate` inherits from, to allow `Value` and `Coalesce` to also use the API hence the `aggregate` name.

In light of this comment do you think `empty_aggregate_value` makes sense?

Reply:
Yup, sorry! 🤕 Nothing else to add then. This is ready.